WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController+Extensions.swift

```diff
@@ -241,6 +246,128 @@ extension NotificationsViewController: WPTableViewHandlerDelegate
 
 
 
+// MARK - Filter Helpers
+//
+extension NotificationsViewController
+{
+    func showFiltersSegmentedControlIfApplicable() {
+        guard tableHeaderView.alpha != WPAlphaZero && shouldDisplayFilters == true else {
```

Comment:
Nice catch!!! i probably meant to write != WPAlphaFull. Fixing right away, thank you!!!